abstract.tex

```diff
@@ -0,0 +1,18 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize many areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
+The LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
+to generate science-ready data products -- processed images, catalogs and alerts. 
+The LSST Science Requirements Document (SRD) defines Key Performance Metrics (KPMs), scalar quantities that measure the quality and scientific fidelity of the LSST data products, together with specifications that represent the capability or accuracy required by the system to achieve the scientific goals. 
+KPMs are defined for a wide range of quantities such as image quality and depth, astrometric and photometric performance, and object recovery completeness. 
+By measuring and tracking KPMs we are able to evaluate trends in algorithmic performance, or compare the performance of one algorithm against another.
+By comparing measured KPMs to specifications we can verify that the LSST data products will meet requirements. 
+In this paper we present \textit{faro}, a framework to efficiently quantify the scientific performance of the LSST data products for data units of varying granularity, ranging from single-detector to full-survey summary statistics, and to persist the results as scalar metric values alongside the input data products.
+We present results using \textit{faro} to characterize the performance of the data products produced on simulated and precursor data sets, and 
+ discuss plans to use faro to verify the performance of the LSST commissioning data products.
+ 
+\noindent \textbf{100 word summary}
+
+\noindent Once operational, up to 20TB of raw imaging data will be collected by Rubin Observatory's Legacy Survey of Space and Time (LSST) per night and processed by the LSST Science Pipelines to produce science-ready data products ? processed images, catalogs and alerts.  To ensure that these data products can enable transformative science with LSST,  stringent requirements have been placed on their quality and scientific fidelity. In this paper we introduce faro,  a framework to efficiently quantify the scientific performance of the LSST data products for data units of varying granularity, ranging from single-detector to full-survey summary statistics. 
```

Comment:
Note: "faro" in the last sentence isn't italicized (or typewriter font).

Reply:
fixed